.travis.yml

```diff
@@ -41,7 +40,22 @@ script:
       pytest -vsx
     fi
 
-matrix:
+jobs:
   include:
-    - python: 3.6
+    - name: internal-ssl test
+      python: 3.6
       env: TEST=internal-ssl
+    - name: python:3.5 + jupyterhub 0.8 + tornado < 5
+      python: 3.5
+      env: JUPYTERHUB=0.8 EXTRA_PIP='tornado<5'
+    - name: python:3.6 + jupyterhub 0.8 + tornado < 5
+      python: 3.6
+      env: JUPYTERHUB=0.8 EXTRA_PIP='tornado<5'
+    - name: python:3.7 + jupyterhub 0.9
+      python: 3.7
+      env: JUPYTERHUB=0.9
```

Comment:
Since Python 3.7 and 3.8 are new scenarios could you test the latest jupyterhub? Or is that going to break something?

Reply:
I believe it is going to fail, because in JupyterHub 1.0, the tests are using `pytest-asyncio` instead of `pytest-tornado`. Dockerspawner uses some of the fixtures JH, so we'll have to switch here too before testing with latest JupyterHubs.

I started working on this [here](https://github.com/jupyterhub/dockerspawner/compare/master...GeorgianaElena:fix_tests) and managed to get it working for JH 1.0, but for 1.1.0 it's failing because of the issue with 1.1.0 docker tags of the `jupyterhub/singleuser` image.

Reply:
Thanks! Let's get this in and deal with the update later.